toolkit/scss/_temporary-message.scss

```diff
@@ -40,6 +40,11 @@
   padding-bottom: 5px;
 }
 
+.temporary-message-banner {
+  @extend %temporary-message-section;
+  margin-top: 15px;
```

Comment:
Will this give us a slightly-too-big whitespace above these banners when there aren't any flashes?

Reply:
My thinking was that flash messages look ok with that margin, so these should too. Here's an example.
![localhost_buyers_direct-award_g-cloud_projects_457356462217937 ipad pro 1](https://user-images.githubusercontent.com/13836290/41413832-30a9b666-6fdc-11e8-8b69-f41a1687748c.png)